vendor/github.com/Microsoft/opengcs/service/gcsutils/remotefs/utils.go

```diff
@@ -43,6 +43,8 @@ func ExportedToError(ee *ExportedError) error {
 		return os.ErrExist
 	} else if ee.Error() == os.ErrPermission.Error() {
 		return os.ErrPermission
+	} else if ee.Error() == io.EOF.Error() {
```

Comment:
Wondering if the `ExportedError` should implement a `causer` interface to make these translations more transparent https://godoc.org/github.com/pkg/errors#hdr-Retrieving_the_cause_of_an_error